sbin/pfmon

```diff
@@ -391,7 +411,7 @@ run all runtasks
 sub runtasks {
     my $mask = POSIX::SigSet->new(POSIX::SIGCHLD());
     sigprocmask(SIG_BLOCK,$mask);
-    while(@REGISTERED_TASKS) {
+    while(@REGISTERED_TASKS && $process) {
```

Comment:
Shouldn't we spawn the worker processes even if they are going to sleep.

That will make it more consistent when looking at the processes list as if it goes from 'slave -> master -> slave', it seems like its going to have them there anyway